src/router.js

```diff
@@ -54,26 +54,26 @@ const routerConfig = {
 			component: () =>
 				import('./views/Statistics.vue')
 		},
-		{
-			path: '/terms',
-			name: 'terms',
-			meta: {
-				group: 'page',
-				keepAliveGoTo: []
-			},
-			component: () =>
-				import('./views/Terms.vue')
-		},
-		{
-			path: '/privacy',
-			name: 'privacy',
-			meta: {
-				group: 'page',
-				keepAliveGoTo: []
-			},
-			component: () =>
-				import('./views/Privacy.vue')
-		},
+		// {
```

Comment:
Keeping this section for later? :D

Reply:
yes, I assume have the same thing with different content, else I will remove it.